py-bns.py

```diff
@@ -42,6 +42,19 @@ def post(self, url, payload):
             e.msg = 'Received a {0} response, instead of 200'.format(status)
             print(e.msg)
 
+    def get(self, url):
+        # headers for the get request
+        get_header = {
+            'content-type': 'application/json',
+            'authorization': 'Bearer {0}'.format(self.access_token)
+        }
+        # set up the url
+        url = self.api_url.format(url)
+        # try requesting a story
+        r = requests.get(url=url, headers=get_header)
+        # return the response
+        return r
```

Comment:
We should use `response.raise_for_status()` here like you did in the `post` function. It's a good way of bubbling up exceptions to the user application.